install.js

```diff
@@ -104,13 +197,30 @@ async function downloadAllTestFiles() {
           console.log(`\tContract From: ${contractDownloadUrl}`);
           console.log(`\tContract To: ${contractDestinationPath}`);
 
-          // Download the contract file
-          await downloadFile(contractDownloadUrl, contractDestinationPath);
-
-          console.log(
-            `\t✅ Successfully downloaded contract ${contractFile}\n\n`
-          );
-          contractSuccessCount++;
+          // Check if contract file exists and ask for confirmation if needed
+          if (fs.existsSync(contractDestinationPath) && !options.force) {
```

Comment:
same as https://github.com/austintgriffith/speedrun-grader/pull/25/files#r2239336737

Reply:
In this case make sense, since it'll only skip the inner loop

Updated https://github.com/austintgriffith/speedrun-grader/pull/25/commits/abb5458c6e6fb3374fcc31fe652ea564a7422aa3